metricbeat/module/elasticsearch/index/index.go

```diff
@@ -80,5 +80,13 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) {
 		return
 	}
 
-	eventsMapping(r, *info, content)
+	if m.XPack {
+		eventsMappingXPack(r, m, *info, content)
```

Comment:
Error ignored in this case?

Reply:
Yeah, see my other comment about why we ignore errors in the x-pack path: https://github.com/elastic/beats/pull/8260#discussion_r215973867